settings/checkstyle/checkstyle.xml

```diff
@@ -6,15 +6,15 @@
 <module name="Checker">
   <property name="charset" value="UTF-8"/>
   <property name="severity" value="error"/>
-  <property name="fileExtensions" value="java, properties, xml"/>
+  <property name="fileExtensions" value="java, kt, properties, xml"/>
```

Comment:
Question: Does Checkstyle ever support Kotlin?

Reply:
I have checked that Kotlin related files are verified with checkstyle.

But the main one seems to be [`ktlint`](https://ktlint.github.io/).

Reply:
I wonder if it is right to apply checkstyle to the kotlin example. 🤔

Reply:
+1 for following ktlint.

Reply:
I applied `ktlint`. However, since it is the [default](https://github.com/pinterest/ktlint#standard-rules), it may be different from the `checkstyle` setting. Do I have to fit it all?

From what I've noticed, it's limited to setting options, so I need to create a [custom ruleset](https://github.com/pinterest/ktlint#creating-a-ruleset).

Reply:
I think it's fine to use the ktlint default. Kotlin is not Java.